src/renderer/runtime.ts

```diff
@@ -499,7 +499,7 @@ class Electron implements Runtime {
   }
   openLink(link: string): void {
     if (link.startsWith('mailto:')) {
-      processOpenQrUrl(link)
+      processMailtoUrl(link)
```

Comment:
Since we're really only dealing with `mailto:` links here I thought its clearer to also address it like that